src/main/java/se/sundsvall/byggrintegrator/api/ByggrFileListResource.java

```diff
@@ -0,0 +1,47 @@
+package se.sundsvall.byggrintegrator.api;
+
+import static org.springframework.http.MediaType.TEXT_HTML_VALUE;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import se.sundsvall.byggrintegrator.service.ByggrIntegratorService;
+import se.sundsvall.dept44.common.validators.annotation.ValidMunicipalityId;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.constraints.NotBlank;
+
+@RestController
+@Validated
+@Tag(name = "File list resource", description = "List files on a neighborhood-notification case")
+@RequestMapping(path = "/{municipalityId}")
+@ApiResponse(
```

Comment:
Should this not be on method level?

also missing a 400 and 500 👯

Reply:
It should/will never produce anything else than a 200 since the ExceptionHandling always returnes an html.